test/bad_dtls_test.c

```diff
@@ -548,8 +548,8 @@ static int test_bad_dtls(void)
 
         if (!TEST_true(send_record(rbio, SSL3_RT_APPLICATION_DATA, tests[i].seq,
                                    &tests[i].seq, sizeof(uint64_t)))) {
-            TEST_error("Failed to send data seq #0x%lx (%d)\n",
-                       tests[i].seq, i);
+            TEST_error("Failed to send data seq #0x%x%08x (%d)\n",
+                       (int)(tests[i].seq >> 32), (int)(tests[i].seq & 0x0FFFFFFFF), i);
```

Comment:
Wasn't it cast to unsigned int initially? Thing is that documentation for "x" reads "unsigned int argument is converted to unsigned hexadecimal." BTW, cast is sufficient, you don't have to mask with 0xFFFFFFFF...